Kernel/Debug.h.in

```diff
@@ -87,6 +87,14 @@
 #cmakedefine01 EXT2_VERY_DEBUG
 #endif
 
+#ifndef FAT_DEBUG
+#cmakedefine01 FAT_DEBUG
+#endif
+
+#ifndef FAT_VERY_DEBUG
+#cmakedefine01 FAT_VERY_DEBUG
+#endif
+
```

Comment:
Having two different log levels seems a bit over the top to me. The important parts of the initialization output (size, drive number, volume id, etc.) as well as some error messages could probably remain unguarded by a debug flag. The rest seems noisy enough (especially the lines on read/write operations) that it could be moved into `FAT_VERY_DEBUG` instead, which could then be the only debug flag, `FAT_DEBUG`.

Reply:
While I agree that it's unnecessary to have two debug flags, but I still believe all the output should be guarded (except the invalid signature error).